packages-native/animation/README.md

```diff
@@ -0,0 +1,332 @@
+## Animation
+
+Bring animations to your Native application by placing content in the animate widget.
+
+#### Usage
+
+-   Place the widget on the page
+-   Place the content inside the content section
+-   Select the animation
+-   Optional set the conditions based on time, visibilty or expression
+
+### Animations
+
+We group the animation effect in 3 categories; entrance, attention and exits;
+
+#### Entrance
+
+Bounce  
+![Bounce entrances](assets/bounce-entrances.gif)
+
+Fading  
+![Fading entrances](assets/fading-entrances.gif)
+
+Sliding  
+![Sliding entrances](assets/sliding-entrances.gif)
+
+Zooming  
+![Zooming entrances](assets/zooming-entrances.gif)
+
+Flip  
+![Flip entrances](assets/flippers.gif)
+
+Light speed  
+![Light speed entrance](assets/light-speed.gif)
+
+#### Attention
+
+![Attention](assets/attention.gif)
+
+#### Exits
+
+Bounce  
+![Bounce exits](assets/bounce-exits.gif)
+
+Zooming  
+![Zooming exits](assets/zooming-exits.gif)
+
+Fading  
+![Fading exits](assets/fading-exits.gif)
+
+Sliding  
+![Sliding exits](assets/sliding-exits.gif)
+
+Flipping  
+![Flip exits](assets/flippers.gif)
+
+Light speed  
+![Light speed exit](assets/light-speed.gif)
+
+### Easing
+
+Easing functions specify the rate of change of a animations over time.
+
+Objects in real life don’t just start and stop instantly, and almost never move at a constant speed. When we open a
+drawer, we first move it quickly, and slow it down as it comes out. Drop something on the floor, and it will first
+accelerate downwards, and then bounce back up after hitting the floor.
+
+For live samples see: https://easings.net/en#
+
+### Styling
+
+To add a defaults style in older Atlas versions, you can add a file `theme/styles/native/app/custom.js` with the
+following content:
+
+```js
+export const com_mendix_widget_native_animation_Animation = (Animation = {
+    container: {
+        // All ViewStyle properties are allowed
+    }
+});
+```
+
+### Design properties
+
+To bring the same design properties as the container widget into an older Atlas versions. You can add to the
+`theme/settings-native.json` the following design properties:
+
+```js
+"com.mendix.widget.native.animation.Animation": [
```

Comment:
Do we have a documentation to link to these properties ? That allows us to not update this README every time there is a change in the design properties.

Reply:
yes, will remove this when its part of Atlas, the widget can live without design props